pkgs/misc/vim-plugins/generated.nix

```diff
@@ -5661,7 +5745,7 @@ final: prev:
       owner = "flazz";
       repo = "vim-colorschemes";
       rev = "fd8f122cef604330c96a6a6e434682dbdfb878c9";
-      sha256 = "1cg8q7w0vgl73aw1b9zz0zh5vw5d2pm8pm54fhfzva4azg56f416";
+      sha256 = "0kpsf6j20fgblc8vhqn7ymr52v2d1h52vc7rbxmxfwdm80nvv3g5";
```

Comment:
this doesn't seem to be right
```
hash mismatch in fixed-output derivation '/nix/store/2r0hrjgnhb8nyf5zh0gn8xwyqxazrg9f-source':
  wanted: sha256:0kpsf6j20fgblc8vhqn7ymr52v2d1h52vc7rbxmxfwdm80nvv3g5
  got:    sha256:1cg8q7w0vgl73aw1b9zz0zh5vw5d2pm8pm54fhfzva4azg56f416
```

Reply:
not sure how that occurred, anyways have kept it as 1cg8q7w0vgl73aw1b9zz0zh5vw5d2pm8pm54fhfzva4azg56f416

Reply:
If ofborg is telling it is wrong it needs to be updated. Also please resolve the merge conflict.

Reply:
rebased and ran `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"` which succeeded

Reply:
```suggestion
      sha256 = "sha256-JhBnyvuKqP0ddKTUi+oVrfBd4Af/pxW4Goe+DfjB6LE=";
```